cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -2145,6 +2145,7 @@ features:
         1: "pipeline "
         _: "pipelines "
     DeployedPipelineView:
+      graphQLMultipleServicesDown: Multiple services ran into issues when trying to retrieve list of pipelines. Please try again later.
```

Comment:
nit: can you validate this message with @leacd ? 

I think its a bit better to say `Multiple services encountered some issues retrieving pipelines....`.

Reply:
Reached out to Lea, waiting on her suggestion

Reply:
updated the message